src/lib/api/contrib/elasticsearch/ESSearchApi.js

```diff
@@ -41,12 +41,9 @@ export class ESSearchApi {
     this.responseSerializer = new responseSerializerCls();
   }
 
-  initAxios(config) {
+  initAxios({ url, ...config }) {
```

Comment:
@slint How is this different than the existing implementation?
Before the code was expecting an object e.g `{url: '/me'}` and this would end up in the non desired behaviour of calling `/me/me` when it was triggering a request. Now with the change passing the passing the same object I think it would lead to the same outcome no? Unless you pass explicitly the `baseURL` parameter which in that case it was already possible before...Do I miss something?

Reply:
Since now `url` is a destructured value from the passed object function argument, it is will not be inside the `...config` object. This means that it won't be passed at all to the `axios.create(...)` call

Reply:
But then you will not be able to pass the `url` parameter at all right?

Reply:
Yup. We have to discuss whether we'll directly pass-through the config to `axios.create(...)`, or if we'll introduce an `axiosConfig` parameter to the `ESSearchApi`/`InvneioSearchApi` class constructor.

Reply:
@ntarocco what do you think? I would be in favor I think of name spacing the axios config like `axiosConfig` and pass it through to axios.

Reply:
Are we discussing this? https://github.com/inveniosoftware/react-searchkit/issues/88
If yes, I should have already replied there :)

Reply:
@ntarocco sorry I missed that. So just to confirm what you are proposing is just get through all the config passed to the axios instance right? In that case I wouldn't destructure the `url` parameter from the config passed and just pass through the config as it is. Then, it would be clear for the users (we should update the docs for that too) that this config is the same with what axios is accepting. @slint WDYT?

Reply:
I would go with an explicit `axiosConfig` parameter for the `InvenioSearchApi` constructor:

https://github.com/inveniosoftware/react-searchkit/blob/375f51db4b3859ddf809f97b513cc0f7e428c544/src/lib/api/contrib/invenio/InvenioSearchApi.js#L15-L16

So something like:

```javascript
export class InvenioSearchApi {
  constructor(serializers, axiosConfig, axiosInterceptors) {
    // NOTE: Would still validate the axiosConfig, but inside "initAxios"
    // this.validateConfig(config);
    this.initSerializers(serializers);
    this.initAxios(axiosConfig, axiosInterceptors);
  }
  ...
}
```